### PR TITLE
#150 Add copy to clipboard to actions

### PR DESCRIPTION
**Pull request by prasadwalvekar**
### Changes

- Add 'Copy to clipboard' to ActionTypes
- Populate TextInput and TextArea fields in dropdown when `onClick` 'Copy to clipboard' is selected
- Show success toast on successful copy

### CODE REVIEW

**Comment by prasadwalvekar** [comment]
@Navaneeth-pk Can you please review?

**Comment by prasadwalvekar** [comment]
Feature loom - https://www.loom.com/share/785a35f230924bf8ae434cfedd14341a

File: frontend/src/Editor/Inspector/EventSelector.jsx [hunk 1]
```diff
@@ -130,6 +130,16 @@ export const EventSelector = ({
                 </div>
               )}
 
+              {definition.actionId === 'copy-to-clipboard' && (
```
Comment (subintp) [inline frontend/src/Editor/Inspector/EventSelector.jsx:133]: Please check below comment

File: frontend/src/_helpers/appUtils.js [hunk 2]
```diff
@@ -111,6 +120,11 @@ function executeAction(_ref, event) {
 
       _ref.setState(newState)
     }
+
+    if (event.actionId === 'copy-to-clipboard') {
```
[the remark this answers is not in the document]
Comment (prasadwalvekar) [reply frontend/src/_helpers/appUtils.js:124]: This was done for consistency as similar approach has been followed for other event actions. 
Can we create a separate issue for this and make changes to all event actions? Will submit a follow up PR.
cc @Navaneeth-pk

[the remark this answers is not in the document]
Comment (Navaneeth-pk) [reply frontend/src/_helpers/appUtils.js:124]: Sounds good 👍

**Comment by prasadwalvekar** [comment]
Replace with select dropdown with input to support dynamic query variables. Loom - 

https://www.loom.com/share/58202ca6c3a547dcb208e175b064b511

**Comment by prasadwalvekar** [comment]
Add copy to clipboard action - 

https://www.loom.com/share/fba10145bb6941529f53642c37850641

**Comment by Navaneeth-pk** [comment]
The label of the field to enter text is " text input ", it should be just " text " @prasadwalvekar

**Pull request merged**